var/spack/repos/builtin/packages/nasm/package.py

```diff
@@ -24,13 +27,37 @@ class Nasm(AutotoolsPackage):
     # https://bugzilla.nasm.us/show_bug.cgi?id=3392461
     patch('https://src.fedoraproject.org/rpms/nasm/raw/0cc3eb244bd971df81a7f02bc12c5ec259e1a5d6/f/0001-Remove-invalid-pure_func-qualifiers.patch', level=1, sha256='ac9f315d204afa6b99ceefa1fe46d4eed2b8a23c7315d32d33c0f378d930e950', when='@2.13.03 %gcc@8:')
 
+    patch('msvc.mak.patch', when='platform=windows')
+
     conflicts('%intel@:14', when='@2.14:',
               msg="Intel 14 has immature C11 support")
 
+    phases = ['configure', 'build', 'install']
```

Comment:
I personally have no problem doing it either way really, but I wonder if straying from convention will result in potential confusion for future developers? If you don't think so I think this and Perl would really benefit from it.

Reply:
This also is partly what prompted https://github.com/spack/spack/pull/27021#issuecomment-1024462191: it might be worth just adding a TODO to split this off into a Windows-specific package repository.